src/FastTags/LexerM.hs

```diff
@@ -0,0 +1,780 @@
+----------------------------------------------------------------------------
+-- |
+-- Module      :  FastTags.LexerTypes
```

Comment:
I know repeating the module name in the comment is semi-standard but I never liked it, and this is one reason why... it's already out of date!  I'd just delete all of them, but if you still like them then at least update

Also it would be nice to have a line or so of overview for the module.